app/helpers/conditions-for-contrib-removal.js

```diff
@@ -0,0 +1,31 @@
+import Ember from 'ember';
+
+/**
+ * conditionsForContribRemoval helper - used to determine if the removing a particular
+ * contributor will still satisfy two conditions 1) @ least one registered admin 2) @ least one
+ * bibliographic contributor
+ *
+ */
+export function conditionsForContribRemoval(params/*, hash*/) {
+    var contributorToRemove = params[0];
+    var contributors = params[1];
+    if (contributors) {
+        var minRegisteredAdmins = false;
+        var minBibliographic = false;
+        contributors.forEach(function(contributor) {
+            if (contributor.id !== contributorToRemove.id) {
+                if (contributor.get('permission') === 'admin' && contributor.get('unregisteredContributor') === null) {
+                    minRegisteredAdmins = true;
+                }
+                if (contributor.get('bibliographic')) {
+                    minBibliographic = true;
+                }
+            }
+        });
+        return minRegisteredAdmins && minBibliographic;
+    } else {
+        return params;
```

Comment:
ok - so I was definitely getting errors before in my console about how certain params couldn't be found, hence the check in all my helpers if the params were there first.  But I am not seeing that anymore so I removed the return of the array data type.